web/frontend/src/pages/ballot/components/Rank.tsx

```diff
@@ -111,13 +111,14 @@ const Rank: FC<RankProps> = ({ rank, answers, language }) => {
               <ul className={rank.ID} {...provided.droppableProps} ref={provided.innerRef}>
                 {Array.from(answers.RankAnswers.get(rank.ID).entries()).map(
                   ([rankIndex, choiceIndex]) => {
-                    if (rank.ChoicesMap.get('en') === undefined) return;
-                    if (language === 'en' && rank.ChoicesMap.has('en'))
-                      return choiceDisplay(rank.ChoicesMap.get('en')[choiceIndex], rankIndex);
-                    else if (language === 'fr' && rank.ChoicesMap.has('fr'))
+                    if (language === 'fr' && rank.ChoicesMap.has('fr'))
                       return choiceDisplay(rank.ChoicesMap.get('fr')[choiceIndex], rankIndex);
                     else if (language === 'de' && rank.ChoicesMap.has('de'))
                       return choiceDisplay(rank.ChoicesMap.get('de')[choiceIndex], rankIndex);
+                    else if (rank.ChoicesMap.has('en'))
```

Comment:
As `en` is the default language, I moved it to the end of the `if` clauses and removed the `language` comparison. So if it's neither `de`, nor `fr`, or if the chosen language is not available, it tries to use `en`.

Reply:
The default `return` (undefined) was replaced by an empty string though. I'm not sure if that's desirable in `tsx` syntax: undefined clearly means there should be no output.

Reply:
eslint complained that `map` wants to have a value to return. If you think `undefined` is the correct return here, I'll `return undefined`.

Reply:
Github bug: I saw your comment twice, even after reloading the page. Now that I deleted the comment once, it's gone twice. Go figure - sorry...

Reply:
Thank you for interpreting my comment correctly - yes for filtering ! 😁